dali/python/nvidia/dali/plugin/tf.py

```diff
@@ -115,6 +128,73 @@ def DALIIterator():
 def DALIRawIterator():
     return _dali_tf
 
+def get_tf_minor_version():
+  return tf.__version__.split('.')[1]
+
+
+class DALIDatasetV2(dataset_ops.DatasetSource):
+  def __init__(
+    self,
+    pipeline = '',
+    batch_size = 1,
+    num_threads = 4,
+    device_id = 0,
+    exec_separated = False,
+    prefetch_queue_depth = 2,
+    cpu_prefetch_queue_depth = 2,
+    gpu_prefetch_queue_depth = 2,
+    shapes = [], 
+    dtypes = []):
+
+    output_classes = tuple(ops.Tensor for shape in shapes)
+
+    self._pipeline = pipeline.serialize()
+    self._batch_size = batch_size
+    self._num_threads = num_threads
+    self._device_id = device_id
+    self._exec_separated = exec_separated
+    self._prefetch_queue_depth = prefetch_queue_depth
+    self._cpu_prefetch_queue_depth = cpu_prefetch_queue_depth
+    self._gpu_prefetch_queue_depth = gpu_prefetch_queue_depth
+    self._shapes = tuple(tf.TensorShape(shape) for shape in shapes)
+    self._dtypes = tuple(dtype for dtype in dtypes)
+
+    self._structure = structure.convert_legacy_structure(
+      self._dtypes, self._shapes, output_classes)
+
+    if get_tf_minor_version() == '14':
+      print('Using TF 14 veriosn')
```

Comment:
```suggestion
      print('Using TF 14 version')
```

Reply:
I removed those strings altogether to keep the output cleaner.